layouts/shortcodes/comic.html

```diff
@@ -1,16 +1,11 @@
-<div class="comic"></div>
+<div class="comic">
     <a href="https://heyzine.com/flip-book/3e66a13901.html">
         <img src="/images/content_images/numpy-comic.png" alt="NumPy Contributor comic book cover">
     </a>
 </div>
 
 <style>
-    .comic {
+    .comic img {
         max-width: 20%;
-        max-height: 20%;

```

Comment:
We are not changing the image's aspect ratio anywhere, so the max-height rule is unnecessary and has no effect. The max-width rule will handle the height automatically.